### PR TITLE
Create and test FeatureGroup

### PR DESCRIPTION
**Pull request by BibMartin**
Include several object in a layer, and then control it into the layer control.

```python
import folium
from folium.map import *
map = folium.Map()
feature_group = FeatureGroup(name='My feature group')
feature_group.add_children(Marker([45,-30],popup=Popup('-30')))
feature_group.add_children(Marker([45, 30],popup=Popup('30')))
map.add_children(feature_group)
map.add_children(folium.map.LayerControl())
map
```

capture du 2015-11-12 12 08 43

### CODE REVIEW

**Comment by ocefpaf** [comment]
Sweet!

**Comment by ocefpaf** [comment]
I am good with that PR and I will be merging it soon unless some else raise something. (I have only 1 stupid pep8 comment.)

BTW: This PR is a perfect example of how the refactoring work you did was worth the effort.
This implementation is concise and efficient thanks to that refactoring.

**Comment by BibMartin** [comment]
Thanks @ocefpaf. Please tell me your PEP8 comment ; I can change it quite quickly.

File: folium/map.py [hunk 1]
```diff
@@ -217,6 +217,34 @@ def __init__(self, tiles='OpenStreetMap', name=None,
         {% endmacro %}
         """)
 
+class FeatureGroup(TileLayer):
+    def __init__(self, name=None, overlay = True):
+        """Create a FeatureGroup layer ; you can put things in it and handle them as a
```
Comment (ocefpaf) [inline folium/map.py]: I guess my comment did not make it in the first try! I have a flaky connection today :unamused: 

Please change `overlay = True` to `overlay=True`

Comment (BibMartin) [inline folium/map.py]: Fixed and rebased

**Comment by ocefpaf** [comment]
Another great addition :tada: 
Thanks @BibMartin

**Comment by jboynyc** [comment]
Yes, this looks great! Thanks.

**Pull request merged**